data-prepper-plugins/otel-logs-source/src/main/java/org/opensearch/dataprepper/plugins/source/otellogs/OTelLogsSource.java

```diff
@@ -77,6 +80,12 @@ public OTelLogsSource(final OTelLogsSourceConfig oTelLogsSourceConfig,
         this.pipelineName = pipelineDescription.getPipelineName();
         this.authenticationProvider = createAuthenticationProvider(pluginFactory);
         this.requestExceptionHandler = new GrpcRequestExceptionHandler(pluginMetrics);
+        this.byteDecoder = new OTelLogsDecoder();
+    }
+
+    @Override
+    public ByteDecoder getDecoder() {
```

Comment:
how is this used ?

Reply:
It is used during pipeline initialization in PipelineTransformer.java